docs/streams/upgrade-guide.html

```diff
@@ -84,7 +84,14 @@
         and can be obtained by calling <code>Topology#describe()</code>.
         An example using this new API is shown in the <a href="/{{version}}/documentation/streams/quickstart">quickstart section</a>.
     </p>
-
+    
+    <p>
+        With the introduction of <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-202+Move+merge%28%29+from+StreamsBuilder+to+KStream">KIP-202</a>
+        the method <code>StreamsBuilder#merge</code> has been deprecated in favor of <code>KStream#merge</code> 
+        The method signature has also been changed: instead of inputting multiple <code>KStream</code>s into the method at the same time, only one <code>KStream</code> will be inputted.
```

Comment:
`has also been change:` -> `was changed, too:` ?

`inputting` -> `providing` ?

`at the same time` -> `at once` ?

`only one` -> `only a single` ?

`will be inputted` -> `is accepted` ?


Also avoid too long lines... Max 120